### PR TITLE
chore: add options to install pydantic with `bentoml[io-json]`

### PR DESCRIPTION
**Pull request by aarnphm**
Add ability to install optional pydantic with `bentoml[io-json]`

Signed-off-by: Aaron Pham <29749331+aarnphm@users.noreply.github.com>

### CODE REVIEW

File: docs/source/concepts/bento.rst [hunk 1]
```diff
@@ -591,7 +607,7 @@ validation, specify the desired variant in the under ``python.packages`` field:
            packages:
            - "bentoml[tracing-zipkin]"
 
-   .. tab-item:: OpenTelemetry Protocol
+   .. tab-item:: OTLP
```
Comment (jjmachan) [inline docs/source/concepts/bento.rst:610]: Why did you change it to the shorter name here?

**Review by jjmachan — APPROVED** [review]

File: docs/source/concepts/bento.rst (hunk 1 above)
Comment (aarnphm) [reply docs/source/concepts/bento.rst:610]: It is actually the name of the protocol, it should be OTLP :)
